File: CurseCS/Model/Cursa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace curse.Model

{
    [Serializable]
    public class Cursa : Entity<long>
    {
        public int capacitate { get; private set; }

        public Cursa(int capacitate)
        {
            Id = 0;
            this.capacitate = capacitate;
        }
        public Cursa(long id, int capacitate)
        {
            Id = id;
            this.capacitate = capacitate;
        }
    }
}

File: Curse - Spring/DomainProject/src/main/java/Domain/Cursa.java
package Domain;

import java.io.Serializable;

public class Cursa extends Entity<Long> implements Serializable {
    public int Capacitate;


    public void setCapacitate(int capacitate) {
        Capacitate = capacitate;
    }

    public int getCapacitate() {
        return Capacitate;
    }

    public Cursa(int capacitate) {
        Capacitate = capacitate;
    }
}
